doc/rados/configuration/pool-pg-config-ref.rst

```diff
@@ -4,13 +4,12 @@
 
 .. index:: pools; configuration
 
-When you create pools and set the number of placement groups (PGs) for each, Ceph
-uses default values when you don't specifically override the defaults. **We
-recommend** overriding some of the defaults. Specifically, we recommend setting
-a pool's replica size and overriding the default number of placement groups. You
-can specifically set these values when running `pool`_ commands. You can also
-override the defaults by adding new ones in the ``[global]`` section of  your
-Ceph configuration file.
+Ceph uses default values to determine how many placement groups (PGs) will be
+assigned to each pool. We recommend overriding some of the defaults.
+Specifically, we recommend setting a pool's replica size and overriding the
+default number of placement groups. You can set these values when running
+`pool`_ commands. You can also override the defaults by adding new ones in the
+``[global]`` section of your Ceph configuration file.
```

Comment:
This is arguably heterodox to the party line of using `pg_autoscaler`

Also, the antecedent of  "by adding new ones"  would seem to be "defaults" which isn't right, would be more like "by adding option settings"

Reply:
https://github.com/ceph/ceph/pull/55419

It took a year, but this part of the documentation is now getting my full attention.